.github/workflows/release.yaml

```
@@ -413,6 +413,20 @@ jobs:
         export TAG=$(curl --silent "https://api.github.com/repos/replicatedhq/kots/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")')
         echo "GORELEASER_PREVIOUS_TAG=${TAG}" >> $GITHUB_ENV
 
+    - uses: sigstore/cosign-installer@v1.2.1
```

This will keep dependabot from spamming us with minor version updates.
```suggestion
    - uses: sigstore/cosign-installer@main
```